README.md

```diff
@@ -1 +1,142 @@
-# hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+# 📗 Table of Contents
+
+- [� Table of Contents](#-table-of-contents)
```

Comment:
- [ ] Table of content inside `table of content` links is redundant and not needed as user won't need to click on this link when they are already on the table of content. I would suggest you remove it from here as well.

Reply:
I'm sorry but I've tried to address this, but every time I remove it keeps returning.